docs/templates/document-templates/render-engine_README.md

```diff
@@ -0,0 +1,44 @@
+# {Name} {#required}
+
+> **NOTE**: Modify the `{Name}` and insert the name of your render engine. Use Heading 1 (H1).
+
+## Overview {#required}
+
+Provide a description of the render engine's functionality - what kind of resources the described render engine are adapted and (if it is) on what it is based, and what it uses (what packages).
+
+## Prerequisites {#required}
+
+> **NOTE**: This section is optional if only required dependency is `@kyma-project/documentation-component`.
+
+List the required packages to use the render engine. 
+
+> **Example**: 
+> - styled-components: ^4.3.0
+
+## Installation {#required}
+
+Explain the steps to install your render engine.
+
+> **Example**: 
```

Comment:
I would remove the example. It doesn't seem to be needed as you can see examples in an existing README.md.